charts/netbox/values.yaml

```
@@ -1499,6 +1512,17 @@ worker:
   ## NOTE: ReadWriteMany PVC(s) are required if replicaCount > 1
   ##
   replicaCount: 1
+  ## Configure PodDisruptionBudget for worker pods
+  ## @param worker.pdb.enabled Enable PodDisruptionBudget for worker pods
+  ## @param worker.pdb.minAvailable Minimum number/percentage of pods that must be available
+  ## @param worker.pdb.maxUnavailable Maximum number/percentage of pods that can be unavailable
+  ## If neither minAvailable nor maxUnavailable is set, a sensible default is applied:
+  ## - Single replica: minAvailable: 1
+  ## - Multiple replicas or HPA enabled: minAvailable: 50%
+  pdb:
+    enabled: false
+    # minAvailable: 1
+    # maxUnavailable: 1
```

When the value is described in the schema, it is better to add default values to avoid misuse, at least to some extent.

```suggestion
    minAvailable: 0
    maxUnavailable: 0
```